base/commandLine.cpp

```diff
@@ -68,6 +69,13 @@ static const char HELP_STRING[] =
 	"  -z, --list-games         Display list of supported games and exit\n"
 	"  -t, --list-targets       Display list of configured targets and exit\n"
 	"  --list-saves=TARGET      Display a list of saved games for the game (TARGET) specified\n"
+	"  -a, --add                Add a game from current or specified directory\n"
+	"                           Use --path=PATH before -a, --add to specify a directory.\n"
+	"  --massadd                Add all games from current or specified directory and all sub directories\n"
```

Comment:
@yinsimei yes, that's essentially the idea. And `--add` would error out if there are multiple games with no `--game` option specified (possibly enumerating them or just advising to use `--detect`)
@bgK good idea, I like it.